sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -230,6 +231,23 @@ statement
     | unsupportedHiveNativeCommands .*?                                #failNativeCommand
     ;
 
+executeImmediate
+    : EXECUTE IMMEDIATE queryParam=executeImmediateQueryParam (INTO LEFT_PAREN targetVariable=multipartIdentifierList RIGHT_PAREN)? (USING params=executeImmediateArgumentSeq)?
```

Comment:
I just double checked that while Snowflake requires USING ( ), Google does has no (. ).
Also PREPARE/EXECUTE does NOT use ( ).
Can we make parens optional for USING and remove them for INTO (or also make them optional)?